xrdp/xrdp_wm.c

```diff
@@ -29,7 +29,10 @@
 #include "log.h"
 #include "string_calls.h"
 
-
+/* Code values used in 'xrdp_mm->code=' settings */
+#define XVNC_SESSION_CODE 0
+#define XRDP_SESSION_CODE 10
+#define XORG_SESSION_CODE 20
 
 /*****************************************************************************/
```

Comment:
Also suggest adding this define to xrdp_types.h to avoid code duplication in xrdp_wm.c:-

```c
// XVNC session has not implemented the touch event for now
#define XRDP_MM_IMPLEMENTS_TOUCH(mm) ((mm)->code != XVNC_SESSION_CODE)
```

Reply:
Done

Reply:
> Also suggest adding this define to xrdp_types.h to avoid code duplication in xrdp_wm.c:-
> 
> ```c
> // XVNC session has not implemented the touch event for now
> #define XRDP_MM_IMPLEMENTS_TOUCH(mm) ((mm)->code != XVNC_SESSION_CODE)
> ```

Done